### DISCUSSION TITLE
【OpenCompass 2.0 CompassRank】How to view the configuration item of the score？

### DISCUSSION THREAD

**Discussion by bullw** [open-compass/opencompass#861] [Q&A]
I can hover over the configuration items on the 1.0 list.
Suspend on the 2.0 leaderboard without any action, unable to view configuration.
How to view configuration items on the 2.0 leaderboard?

![image](https://github.com/open-compass/opencompass/assets/46083912/e44c34df-4f93-4387-92a6-dbfce0c3380b)

**Comment by zh190920** [comment]
1.0的配置对吗，有的base模型确用了chat的数据集模板。如果你有答案了，请告诉我一下哈
